Library/Homebrew/cmd/update-report.rb

```diff
@@ -249,6 +249,20 @@ def updated?
   def migrate_tap_migration
     report[:D].each do |full_name|
       name = full_name.split("/").last
+      if tap == "caskroom/cask"
+        next unless (dir = HOMEBREW_REPOSITORY/"Caskroom"/name).exist?
```

Comment:
The assignment to `dir` seems unnecessary. This variable is not used in the rest of this code block, unless I'm overlooking something.